src/rgw/rgw_main.cc

```diff
@@ -364,6 +376,25 @@ int radosgw_Main(int argc, const char **argv)
     derr << "Couldn't init storage provider (RADOS)" << dendl;
     return EIO;
   }
+
+#ifdef WITH_RADOSGW_DBSTORE
+  if (config_store == "dbstore") {
```

Comment:
@dang @cbodley .. this is additional change (*temporary* till dbstore CLIs are available) to create testid user on rgw startup on dbstore.  Please let me know if these temp changes here are fine or is it better to add it in sal_dbstore*/dbstore* layer instead (probably while creating .db connection)?

Reply:
I'm okay with this.

Reply:
sure. my only feedback here is that it doesn't need to be in rgw_main.cc; maybe `StoreManager::init_storage_provider()` in rgw_sal.cc is a better place for it?